src/oge/load_data.py

```diff
@@ -143,7 +143,11 @@ def load_cems_ids() -> pd.DataFrame:
             filters=[["year", "==", year]],
             columns=["plant_id_epa", "plant_id_eia", "emissions_unit_id_epa"],
         ).drop_duplicates()
+        cems_id_year = apply_dtypes(cems_id_year)
+        # update the plant_id_eia column using manual matches
+        cems_id_year = update_epa_to_eia_map(cems_id_year, year)
```

Comment:
This was moved up to apply dytpes and EIA mappings on a year-by-year basis, rather than at the end.